studiopress/atomic-blocks
# Improve ABP onboarding page

**Summary of change:**
Update onboarding page for ABP. This update makes the analytics setting more obvious, recommends Genesis block-powered themes, and updates the resources and links.

**How to test:**
The page will show the updated content when it detects ABP. You can set the `atomic_blocks_is_pro` function to false to see what it would look like with AB free.

**Suggested Changelog Entry:**
Update Getting Started page with a better focus on Atomic Blocks Pro resources.

@mindctrl The analytics box here will need to be connected to the settings. Thanks!
